### PR TITLE
CSHARP-2313: Fix Sandcastle doc comment warnings

### PR DESCRIPTION
**Pull request by vincentkam**

### CODE REVIEW

File: Docs/Api/CSharpDriverDocs.shfbproj [hunk 1]
```diff
@@ -72,7 +72,7 @@
       <NamespaceSummaryItem name="MongoDB.Driver.Core.WireProtocol.Messages.Encoders.BinaryEncoders" isDocumented="True">The MongoDB.Driver.Core.WireProtoco.Messages.Encoders.BinaryEncoders namespace contains classes related to binary message encoders.</NamespaceSummaryItem>
       <NamespaceSummaryItem name="MongoDB.Driver.Core.WireProtocol.Messages.Encoders.JsonEncoders" isDocumented="True">The MongoDB.Driver.Core.WireProtoco.Messages.Encoders.JsonEncoders namespace contains classes related to JSON message encoders.</NamespaceSummaryItem>
     </NamespaceSummaries>
-    <CopyrightText>Copyright &amp;#169%3b 2010-2015 MongoDB Inc.</CopyrightText>
+    <CopyrightText>Copyright &amp;#169%3b 2010&amp;ndash%3bpresent MongoDB Inc.</CopyrightText>
```
Comment (vincentkam) [inline Docs/Api/CSharpDriverDocs.shfbproj:75]: Swapped out the hyphen for an endash.

File: Docs/Api/CSharpDriverDocs.shfbproj [hunk 2]
```diff
@@ -1,7 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
   <PropertyGroup>
-    <SHFBROOT Condition=" '$(SHFBROOT)' == '' ">$(MSBuildThisFileDirectory)..\..\packages\EWSoftware.SHFB.2017.1.28.0\Tools\</SHFBROOT>
```
Comment (vincentkam) [inline Docs/Api/CSharpDriverDocs.shfbproj:4]: We shouldn't need to manually specify where SHFB is.

**Review by rstam — APPROVED** [review]
Awesome that you solved the <inheritdoc> issue!!!

**Comment by vincentkam** [comment]
Thanks Robert! 😺 
Rebased, squashed, pushed.

**Pull request merged**